src/app/vertical-nav/unstructured-routes/unstructured-routes.html

```diff
@@ -0,0 +1,198 @@
+<!--
+  ~ Copyright (c) 2017 VMware, Inc. All Rights Reserved.
+  ~ This software is released under MIT license.
+  ~ The full license information can be found in LICENSE in the root directory of this project.
+  -->
+
+<div class="clr-example">
+    <div class="main-container">
+        <header class="header header-6">
+            <div class="branding">
+                <a href="#">
+                    <clr-icon shape="vm-bug"></clr-icon>
+                    <span class="title">Project Clarity</span>
+                </a>
+            </div>
+            <form class="search">
+                <label for="search_input">
+                    <input id="search_input" type="text" placeholder="Search for keywords...">
+                </label>
+            </form>
+            <div class="settings">
+                <a href="javascript://" class="nav-link nav-icon">
+                    <clr-icon shape="cog"></clr-icon>
+                </a>
+            </div>
+        </header>
+        <div class="content-container">
+            <div class="content-area">
+                <div class="example-options">
+                    <h6>Options</h6>
+                    <div class="radio-inline">
+                        <input type="radio" name="vertNavRadios" id="rad1" [(ngModel)]="option" value="lazy">
+                        <label for="rad1">Lazy Loading</label>
+                    </div>
+                    <div class="radio-inline">
+                        <input type="radio" name="vertNavRadios" id="rad2" [(ngModel)]="option" value="no-lazy">
+                        <label for="rad2">No Lazy Loading</label>
+                    </div>
+                </div>
+                <div class="controls">
+                    <h6>Controls</h6>
+                    <button class="btn" (click)="toggleNav()">Toggle Nav</button>
+                    <button class="btn" (click)="toggleGroup()">Toggle Group</button>
+                </div>
+                <router-outlet></router-outlet>
+            </div>
+            <clr-vertical-nav *ngIf="option === 'lazy'"
```

Comment:
Added notes for the both the cases. Fixed

![image](https://user-images.githubusercontent.com/1426805/29364756-8fa84eba-8262-11e7-9547-decda686c512.png)

![image](https://user-images.githubusercontent.com/1426805/29364766-99cf37d2-8262-11e7-801a-e2212e021082.png)